client/deployments/client.go

```diff
@@ -72,26 +72,60 @@ const (
 	artifactUploadURL  = "/api/management/v1/deployments/artifacts"
 	artifactsListURL   = artifactUploadURL
 	artifactsDeleteURL = artifactUploadURL
+	directUploadURL    = "/api/management/v1/deployments/artifacts/directupload"
 )
 
 type Client struct {
 	url               string
 	artifactUploadURL string
 	artifactsListURL  string
 	artifactDeleteURL string
+	directUploadURL   string
 	client            *http.Client
 }
 
+type Link struct {
+	Uri      string            `json:"uri"`
+	Expire   time.Time         `json:"expire,omitempty"`
+	Method   string            `json:"method,omitempty"`
+	Header   map[string]string `json:"header,omitempty"`
+	TenantID string            `json:"-"`
+}
```

Comment:
I see this is a direct copy from the PR in deployments. However, only 3 parameters are needed: `uri`, `header` and `id`. The rest can be ignored.